### ISSUE TITLE
How to build the example?

### ISSUE THREAD

**Issue by ahmedk92** [FolioReader/FolioReaderKit#252]
I'm having hard time building the example. I'm using Xcode 8.3.3.

It complains about converting Swift code. And after conversion, it build successfully but the simulator just hangs.

**Comment by ahmedk92** [comment]
I found a temporary solution; that is to use the Swift 2.3 branch. But of course should be opened with an Xcode version that supports it (e.g. 8.2 like I did).

I followed the instructions [here](https://github.com/FolioReader/FolioReaderKit/issues/129). And had to replace the pod line in the example from:
`pod 'FolioReaderKit', path: '../'`
to:
`pod 'FolioReaderKit', :git => 'https://github.com/FolioReader/FolioReaderKit.git', :branch => 'swift23'`
